cmd/nerdctl/ps.go

```diff
@@ -385,3 +387,57 @@ func getContainerSize(ctx context.Context, client *containerd.Client, c containe
 
 	return fmt.Sprintf("%s (virtual %s)", progress.Bytes(containerSize).String(), progress.Bytes(imageSize).String()), nil
 }
+
+// TODO: this code should be remove to a common package after the refactor completed
+type snapshotKey string
```

Comment:
Is this `snapshotKey` code the same as the code in `pkg/cmd/image/list.go`

Reply:
Yes, the duplicate code will be remove when I refactor the ps command